### DISCUSSION TITLE
Error: Could not import 'superset.exe'. on windows 10 with apache-superset 3.1.1

### DISCUSSION THREAD

**Discussion by duniang818** [apache/superset#27192] [Q&A / Help]
Error: Could not import 'superset.exe'. on windows 10 with apache-superset 3.1.1
error with run command: 
superset db upgrade

**Comment by sfirke** [comment]
Superset is not made to run on Windows.  As far as I know the only option would be to run Windows Subsystem for Linux 2 (WSL2) and then deploy Superset with docker compose.
